src/assets/styles/react-grid-layout.scss

```diff
@@ -0,0 +1,44 @@
+// Modified version of node_modules/react-grid-layout/css/style.css
+
+.react-grid {
+	&-layout {
+		position: relative;
+		transition: height 200ms ease;
+	}
+
+	&-item {
+		transition: all 200ms ease;
+
+		&.react-draggable-dragging {
+			transition: none;
+			z-index: 3;
+			will-change: transform;
+		}
+
+		&.react-grid-placeholder {
+			z-index: 2;
+			user-select: none;
+			pointer-events: none;
+		}
+	}
+}
+
+// Custom handle for dragging layout items
+.draggable-handle {
+	position: absolute;
+	content: ' ';
```

Comment:
Why do we need this?

Reply:
It is used on mobile to implement dragging bookmarks. Without it, a library that does Drag & Drop does not handle clicking on a bookmark and always thinks that a user wants to move it.
It is disabled on a desktop because it does not have this problem, but you can test it by setting up a mobile user agent.

Here's a demo of how it looks:
![video_2024-07-08_21-43-42](https://github.com/kse-web-development-practice/shcherbakov-fluent-start-page/assets/29505891/a8cf79a0-a319-49dc-a483-cfd9e8e9f06b)

Reply:
Speaking about `content` property, you are right, it is not needed. Removed it.
Sorry for my previous comment. Github highlights not 1 but 4 lines of code and I thought you were writing about the whole `.draggable-handle` class.

<img src="https://github.com/kse-web-development-practice/shcherbakov-fluent-start-page/assets/29505891/844673bb-23f1-48d0-baad-d77bf4e13841" width="400" />